scripts/synapse_port_db

```diff
@@ -820,7 +820,7 @@ class Porter(object):
                     "ALTER SEQUENCE events_stream_seq RESTART WITH %s", (next_id,)
                 )
 
-            txn.execute("SELECT -MIN(stream_ordering) FROM events")
+            txn.execute("SELECT GREATEST(-MIN(stream_ordering), 1) FROM events")
             curr_id = txn.fetchone()[0]
             if curr_id:
```

Comment:
when would this be falsey?

Reply:
Ah, yeah now we have a `GREATEST` it will never be null :+1: